app/components/shared/courses/financial_support/fees_and_financial_support_component/view.html.erb

```diff
@@ -19,7 +19,7 @@
     <% end %>
   <% end %>
 
-  <% if financial_support.present? %>
+  <% if financial_support.present? && !course.recruitment_cycle.after_2024? %>
```

Comment:
It's not possible to view courses from previous cycles - how do you feel about completely removing this from the view?

Reply:
I was thinking that, but also I was thinking that providers are worry for tomorrow and I am sign off soon too. I can try give it a go for 10 minutes and see if is only few specs breaks to change.

Reply:
Also the financial support is all over the data (on enrichments) and were copied over to this cycle. I think we should amend somehow as well.

Reply:
> It's not possible to view courses from previous cycles - how do you feel about completely removing this from the view?

This is done and pushed.